src/main/kotlin/it/pagopa/ecommerce/helpdesk/services/EcommerceService.kt

```diff
@@ -18,7 +19,11 @@ class EcommerceService {
     ): Mono<SearchTransactionResponseDto> {
         logger.info("[helpDesk ecommerce service] searchTransaction method")
         return ecommerceSearchTransactionRequestDto
-            .doOnNext { logger.info("Search type: ${it.type}") }
+            .doOnNext {
+                logger.info("Search type: ${it.type}")
+                val transactionEvent = TransactionActivatedEvent()
```

Comment:
typo ?

Reply:
Yes, I was just checking that ecommerce commons library was effectively added :D